php/libraries/FeedbackMRI.class.inc

```diff
@@ -57,34 +57,13 @@ class FeedbackMRI
         }
     }
 
-    /**
-     * Clears all predefined comments from the object
-     * (used prior to re-setting the list of predefined comments)
-     *
-     * @return bool true if operation succeeded
-     */
-    function clearPredefinedComments()
-    {
-        // start the query
-        $query = "DELETE FROM feedback_mri_comments WHERE ".$this->_buildWhere();
-
-        // limit to predefined comments
-        $query .= "AND PredefinedCommentID IS NOT NULL";
-
-        // run the query
-        $success = mysql_query($query);
```

Comment:
This function has not existed in PHP since before 7.0.0 and LORIS has never complained so... I think it's safe to get rid of this function altogether.